pyproject.toml

```diff
@@ -39,6 +39,7 @@ dependencies = [
     "pyyaml",
     "yabox",
     "joblib",
+    "sesame@git+https://github.com/phoebe-p/sesame.git"
```

Comment:
I'm not entirely sure, but I think that `pypi` do not allow dependencies that live in GitHub, only in `pypi` itself. I suspect that if Sesame is added this way, the deployment of Solcore will fail :( .

Reply:
I am actually having a hard time figuring out if this is possible (it definitely used to not be possible, but most of the threads I can find are a few years old and discussing setup.py projects, not pyproject.toml, so I wonder if anything has changed since then). It works in the build system on GitHub Actions and on my computer (by running pip install) but that does not mean that PyPI will accept it. I will make sure to test this if I can't figure it out by uploading to TestPyPI, and if it doesn't work find another solution.

Reply:
You were right that this is not allowed. I've uploaded the version of Sesame I was using to PyPI so it can be installed normally.